Add logger for AJAX middleware #1353

Slange-Mhath opened the pull request:
This commit will introduce a logger which writes errors
into the log file and thus fixes the issue [1353](https://github.com/archivematica/Issues/issues/1353)

sevein commented on src/dashboard/src/middleware/common.py:
```diff
@@ -27,18 +28,29 @@
 from django.utils.deprecation import MiddlewareMixin
 from shibboleth.middleware import ShibbolethRemoteUserMiddleware
 
+
```
We don't need this extra line.

sevein commented on src/dashboard/src/middleware/common.py:
```diff
@@ -27,18 +28,29 @@
 import elasticsearch
 
 
+logger = logging.getLogger(__name__)
```
Please use this form:

    logger = logging.getLogger("archivematica.dashboard")

Otherwise it'll be given a name we don't want to use.

sevein reviewed (changes requested):
Thank you, just a couple of minor requests.

Commit:
Add logger to log exceptions in AJAX middleware

This commit will introduce a logger which writes errors
into the log file and thus fix the issue 1353.

sevein reviewed (approved):
Thank you!
